Sources/UInt128.swift

```diff
@@ -276,10 +276,10 @@ extension UInt128 : FixedWidthInteger {
             lowerUpperBits.overflowCount)
 
         // Bring the 64bit unsigned integer results together into a high and low 128bit unsigned integer result.
-        return (high: .init(upperBits: upperUpperBits.truncatedValue,
-                            lowerBits: lowerUpperBits.truncatedValue),
-                low: .init(upperBits: upperLowerBits.truncatedValue,
-                           lowerBits: lowerLowerBits.truncatedValue))
+        return (.init(upperBits: upperUpperBits.truncatedValue,
```

Comment:
As this method is quite long, I would like to keep the tuple name's present so you don't have to scroll or rely on IDE hints.